### DISCUSSION TITLE
How can I remove the "blue" arrow above my path?

### DISCUSSION THREAD

**Discussion by fhbash** [sindresorhus/pure#661] [General]
Hey y'all.. I'm testing Pure, and so far I love it.. but, there's something it's frustrating to me, this:
![image](https://github.com/sindresorhus/pure/assets/2904206/07f0aa93-2778-451e-897d-6f98cff8c8b5)

How can I remove this "blue" arrow, for each line?

Thanks!

**Comment by sindresorhus** [comment]
That comes from the Terminal app, not Pure.
